transcom/mymove
# B-20291:Customer review page errors int

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20291)

## Summary
Dan requested that if there is no Transportation Office office then show "To change these fields, contact your local PPPO office".

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the MilMove as customer and create a move using a duty location that does not have a Transportation Office OR use a move that already has this circumstance
2. Login to OFFICE as a TOO and approve the move
3. Go back to the MilMove customer and select Review your Request
4. If there move has no Transportation office it will should say "To change these fields, contact your local PPPO office". If it does then it should show the original message with Transportation office name and number.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
![image](https://github.com/user-attachments/assets/ca71fa86-cf35-42ce-a2b5-4ef1119a54a6)
